06osdeps_from_source.autobuild

```diff
@@ -14,5 +14,9 @@ in_flavor 'master', 'stable' do
     end
     Autoproj.add_osdeps_overrides 'opencv', :package => 'external/opencv'
     remove_from_default 'external/opencv'
+
+    cmake_package 'external/poco'
+    Autoproj.add_osdeps_overrides 'poco', :package => 'external/poco'
```

Comment:
we decided a while ago (which you cannot know) that the external/ folder should not be used anymore.
external poco is used by class_loader. I recomment to place the poco inside tools/